mobile-widgets/qml/TemplateLine.qml

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: GPL-2.0
+import QtQuick 2.11
+import QtQuick.Controls 2.4
+import QtQuick.Layouts 1.11
+
+Rectangle {
+	color: "black"
+	height: 1
+	width: parent.width -20
```

Comment:
absolute offsets tend to be a problem - we have users using this on screens that vary widely in width. So 20 pixels could be almost 10% of the width, or only 1% of the width. We try to express width in terms of `Kirigami.Units.gridUnit` (or `...smallSpacing` and `...largeSpacing` - these are all multiples of each other)

Reply:
I think I got it. However the -20 was a real bug, width = parent.width. But I will try to remember this for future changes.